embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/TelemetryAttributes.kt

```diff
@@ -0,0 +1,27 @@
+package io.embrace.android.embracesdk.arch.schema
+
+import io.embrace.android.embracesdk.internal.spans.toEmbraceAttributeName
+import io.embrace.android.embracesdk.session.properties.EmbraceSessionProperties
+
+/**
+ * Object that aggregates various attributes and returns a [Map] that represents the values at the current state
+ */
+internal class TelemetryAttributes(
```

Comment:
Will this be used by spans or just logs?

Reply:
it is only used by Logs now but can be used by spans and span events too. After this merges, I'll convert the other ones